releases/stellar/2025-01-GMP-v1.0.0.md

```diff
@@ -0,0 +1,217 @@
+# STELLAR GMP v1.0.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @ahramy <ahram@interoplabs.io>       |
+| **Deployment** | @RiceAndMeet <steven@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [Crates](https://crates.io/users/interoplabs-ci)
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases)
+
+## Background
+
+Changes in the release:
+
+1. This is the v1.0.0 initial GMP release.
+
+## Deployment
+
+Create an `.env` config.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=stellar-2024-q4
+```
+
+```bash
+# Deploy Gateway
+node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0
+
+# Instantiate stellar VotingVerifier
+node ./cosmwasm/deploy-contract.js instantiate -c VotingVerifier -n stellar-2024-q4 --fetchCodeId --instantiate2
```

Comment:
Use .env instead of hardcoding the chain name. Also, the chain name will be `stellar` on mainnet, and these instructions should work generically